src/checker/basemodels.py

```diff
@@ -420,8 +420,10 @@ def run_checks(solution, env, run_all):
             else:
                 # make non passed result
                 # this as well as the dependency check should propably go into checker class
+                # TODO: Move code to checker class ?
                 result = checker.create_result(env)
-                result.set_log("Checker konnte nicht ausgeführt werden, da benötigte Checker nicht bestanden wurden.")
+                #result.set_log("Checker konnte nicht ausgeführt werden, da benötigte Checker nicht bestanden wurden.")
```

Comment:
```suggestion
```

This comment isn't required any longer, isn't it?

Reply:
There exists an issue covering an idea of internationalising messanges : https://github.com/KITPraktomatTeam/Praktomat/issues/215 : I thought not to loose the "original german" message is a nice idea, so I kept it as comment.